devicetypes/smartthings/cree-bulb.src/cree-bulb.groovy

```diff
@@ -66,6 +67,12 @@ def parse(String description) {
     def resultMap = zigbee.getEvent(description)
     if (resultMap) {
         sendEvent(resultMap)
+        // Temporary fix for the case when Device is OFFLINE and is connected again
+        if (state.lastActivity == null){
+            state.lastActivity = now()
+            sendEvent(name: "deviceWatch-lastActivity", value: state.lastActivity, description: "Last Activity is on ${new Date((long)state.lastActivity)}", displayed: false, isStateChange: true)
```

Comment:
Yeah, however it ran into situations (first time retrieving state.lastActivity) where 
`7ec9ee27-0432-48c7-acf9-e9cfed0c2220 3:52:55 PM: error groovy.lang.GroovyRuntimeException: Could not find matching constructor for: java.util.Date(java.math.BigInteger) @ line 102`